components/hab/src/cli.rs

```diff
@@ -1092,6 +1092,8 @@ fn sub_pkg_download() -> App<'static, 'static> {
             "Target architecture to fetch. E.g. x86_64-linux. Overridden if architecture is specified in toml file")
     (@arg VERIFY: --verify
             "Verify package integrity after download (Warning: this can be slow)")
+    (@arg IGNORE_MISSING_SEEDS: --("ignore-missing-seeds")
```

Comment:
I'm not really happy with this name, but I didn't want to say `ignore-missing-packages` to differentiate between failures resolving seed files, and unable to download the expanded dependency list.